### PR TITLE
Add documentation of ttl option in Readme

### PR DESCRIPTION
**Pull request by ziaulrehman40**
As discussed in #51

### CODE REVIEW

**Comment by ziaulrehman40** [comment]
Documentation update from discussion in #51

File: README.md [hunk 1]
```diff
@@ -116,6 +116,19 @@ end
 if you are using dynamic limit/period options. Otherwise you risk getting into
 some trouble.
 
+### Timt To Live(ttl)
```
Comment (ixti) [inline README.md]: Typo: *Tim**t***

Also please make *ttl* uppercase.

Comment (ziaulrehman40) [reply README.md]: Done.

**Comment by ziaulrehman40** [comment]
@ixti Any blocker on this PR?

**Comment by ixti** [comment]
@ziaulrehman40 no blockers - I will process this PR on weekend.

**Comment by ixti** [comment]
Merged via https://github.com/sensortower/sidekiq-throttled/commit/5a195c11cd590db2171ea52adae5aacd3589b1fb

**Pull request closed without merging**